experiments/evals/run_uncheatable_evals.py

```diff
@@ -0,0 +1,206 @@
+# Copyright 2025 The Marin Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from experiments.llama import llama_3_2_1b as llama_3_2_1b_config, llama3_tokenizer, llama_8b
+from marin.evaluation.log_probs import default_lm_log_probs
+from marin.execution.executor import executor_main
+from marin.processing.tokenize.data_configs import mixture_for_evaluation
+
+from dataclasses import dataclass
+from levanter.models.llama import LmConfig
+from levanter.models.olmo import Olmo2Config
+from experiments.evals.resource_configs import SINGLE_TPU_V5p_8_FULL
+from marin.execution.executor import ExecutorStep
+from experiments.models import (
+    llama_3_1_8b,
+    olmo_2_base_8b,
+    olmo_2_base_32b,
+    marin_8b_base,
+    llama_3_2_1b as llama_3_2_1b_model,
+    qwen3_0_6b,
+    qwen3_1_7b,
+    qwen3_4b,
+    qwen3_8b,
+    qwen3_0_6b_base,
+    qwen3_1_7b_base,
+    qwen3_4b_base,
+    qwen3_8b_base,
+    qwen3_32b,
+)
+from experiments.qwen3 import (
+    qwen3_0_6b as qwen3_0_6b_config,
+    qwen3_1_7b as qwen3_1_7b_config,
+    qwen3_4b as qwen3_4b_config,
+    qwen3_8b as qwen3_8b_config,
+    qwen3_32b as qwen3_32b_config,
+    marin_32b as marin_32b_config,
+)
+from experiments.uncheatable_eval import uncheatable_eval_tokenized
+
+UNCHEATABLE_EVAL_TO_FILE_PATTERN = {
+    # "wikipedia_arabic": "wikipedia_arabic_*.jsonl.gz",
+    "wikipedia_english": "wikipedia_english_*.jsonl.gz",
+    # "wikipedia_french": "wikipedia_french_*.jsonl.gz",
+    # "wikipedia_german": "wikipedia_german_*.jsonl.gz",
+    # "wikipedia_japanese": "wikipedia_japanese_*.jsonl.gz",
+    # "wikipedia_spanish": "wikipedia_spanish_*.jsonl.gz",
+    "github_python": "github_python_*.jsonl.gz",
+    "github_cpp": "github_cpp_*.jsonl.gz",
+    "bbc_news": "bbc_news_*.jsonl.gz",
+    "arxiv_physics": "arxiv_physics_*.jsonl.gz",
+    "arxiv_computer_science": "arxiv_computer_science_*.jsonl.gz",
+    # "ao3_chinese": "ao3_chinese_*.jsonl.gz",
+    "ao3_english": "ao3_english_*.jsonl.gz",
+}
+
+marin_32b_base = (
+    "gs://marin-us-central1/checkpoints/tootsie-32b-cooldown-bison-adamc/hf/tootsie-32b-cooldown-bison-adamc/step-192000"
+)
+
+olmo_7b = Olmo2Config(
+    seq_len=4096,
+    hidden_dim=4096,
+    intermediate_dim=11008,
+    num_heads=32,
+    num_kv_heads=32,
+    num_layers=32,
+)
+
+olmo_32b = Olmo2Config(
```

Comment:
Is this defined elsewhere?  It seems like we should define this once and import it since this references to the actual olmo2 model, not the architecture